drake/doc/development_on_osx.rst

```diff
@@ -0,0 +1,41 @@
+*****************************************
+Drake Development on OS X
+*****************************************
+
+This page contains software development notes that are specific to OS X.
+
+Installing pygame
+===================================
+
+`pygame <http://pygame.org>`_ is used by ``drake/examples/Cars/SteeringCommandDriver.py``.
+
+Install Mercurial:
+
+    brew install mercurial
+
+Install pygame:
+
+    pip install hg+http://bitbucket.org/pygame/pygame
+
+Ensure your ``PYTHONPATH`` environment variable includes Drake's site-package and dist-packages directories:
+
+    export PYTHONPATH=$PYTHONPATH:[DRAKE_ROOT]/build/lib/python2.7/site-packages:[DRAKE_ROOT]/build/lib/python2.7/dist-packages
+
+
+
+Running gitk
+============
```

Comment:
the notes as written sound like you're listing these as the recommended tools.   perhaps prepend some text which just indicates that these have been useful to some?